packages/wallets/src/components/WalletsPercentageSelector/WalletsPercentageSelector.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useCallback, useMemo } from 'react';
+import WalletsPercentageSelectorBlock from './WalletsPercentageSelectorBlock';
+import './WalletsPercentageSelector.scss';
+
+const percentageBlockCount = 4;
+const percentageBlockSize = 100 / percentageBlockCount;
+
+const percentageSelectorOptions = [...Array(percentageBlockCount).keys()].map(index => ({
+    label: index + 1 === percentageBlockCount ? 'All' : `${((index + 1) * 100) / percentageBlockCount}%`,
+    percentage: ((index + 1) * 100) / percentageBlockCount,
+}));
```

Comment:
```suggestion
const percentageBlockCount = 4;
const percentageBlockSize = 100 / percentageBlockCount;

const percentageSelectorOptions = [...Array(percentageBlockCount).keys()].map(index => ({
    label: index + 1 === percentageBlockCount ? 'All' : `${((index + 1) * 100) / percentageBlockCount}%`,
    percentage: ((index + 1) * 100) / percentageBlockCount,
}));
```
A few questions:
- Will this block count ever change? Is there a plan to change it?
If yes, make this`blockCount` as a prop instead of a hard coded constant and move the  `percentageSelectorOptions ` to the component. This way user of this component can modify their block size and it would be flexible to use in the future without too much rewriting.

- You will always select per block correct? Which means in 25% increments?

Reply:
Yes, those are always the same (4 blocks 25% each)